database_mockup/src/animalkeeping/ui/PersonsView.java

```diff
@@ -103,6 +103,14 @@ public void initialize(URL location, ResourceBundle resources) {
             }
         });
         controls.getChildren().add(deleteLabel);
+
+        asUserLabel = new ControlLabel("add as database user", true);
+        deleteLabel.setOnMouseClicked(event -> {
+            if(event.getButton().equals(MouseButton.PRIMARY)){
+                addUser();
+            }
+        });
+        controls.getChildren().add(deleteLabel);
     }
 
 
```

Comment:
add in line 121 
````
asUserLabel.setDisable(p == null);
````
We may want to check whether this user has already been added